explorer/interpreter/value.cpp

```diff
@@ -137,6 +183,20 @@ static auto GetMember(Nonnull<Arena*> arena, Nonnull<const Value*> v,
   }
 }
 
+static auto GetMember(Nonnull<Arena*> arena, Nonnull<const Value*> v,
+                      const FieldPath::Component& field,
+                      SourceLocation source_loc, Nonnull<const Value*> me_value)
+    -> ErrorOr<Nonnull<const Value*>> {
+  switch (field.member()->kind()) {
+    case MemberKind::PositionalMember:
+      return GetPositionalMember(v, field, source_loc);
+    case MemberKind::NominalMember:
+      return GetNamedMember(arena, v, field, source_loc, me_value);
+    default:
+      return GetSpecialMember(v, field, source_loc);
```

Comment:
Mhh... like casting it to `TupleValue` and `NominalClassValue`? Yeah, that'd work currently. This might be overfitting, given this is meant to be flexible for other uses too, but I don't think we have any others currently for Positional (and Base ofc).

Reply:
Done a pass in that direction